lib/ditto.ts

```diff
@@ -169,11 +176,16 @@ const executeCommand = async (
     case "project remove": {
       return removeProject();
     }
+    case "component-folders": {
+      return showComponentFolders();
+    }
     case "generate-suggestions": {
       return generateSuggestions({
         ...(options.directory ? { directory: options.directory } : {}),
         ...(options.files ? { files: options.files } : {}),
-        ...(options.componentFolder ? { componentFolder: options.componentFolder } : {}),
+        ...(options.componentFolder
+          ? { componentFolder: options.componentFolder }
+          : {}),
```

Comment:
nit: for this object, is this equivalent?
```js
return generateSuggestions({
  directory: options.directory,
  files: options.files,
  componentFolder: options.componentFolder
});
```

I think the type of each property is`Value | undefined` in either implementation. I guess you could even just do this but maybe it's too far haha
```js
return generateSuggestions(options);
```